#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_sourced_scripts.dpatch by  <jblache@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Remove the shebang line from openserctl.* scripts which aren't meant
## DP: for direct execution (sourced by openserctl)

@DPATCH@
diff -urNad openser-1.1.0~/scripts/openserctl.base openser-1.1.0/scripts/openserctl.base
--- openser-1.1.0~/scripts/openserctl.base	2006-07-04 19:49:58.000000000 +0200
+++ openser-1.1.0/scripts/openserctl.base	2006-07-12 14:24:41.880689270 +0200
@@ -1,4 +1,3 @@
-#!/bin/sh 
 #
 # $Id$
 #
diff -urNad openser-1.1.0~/scripts/openserctl.ctlbase openser-1.1.0/scripts/openserctl.ctlbase
--- openser-1.1.0~/scripts/openserctl.ctlbase	2006-07-04 19:49:58.000000000 +0200
+++ openser-1.1.0/scripts/openserctl.ctlbase	2006-07-12 14:24:39.197440379 +0200
@@ -1,4 +1,3 @@
-#!/bin/sh 
 #
 # $Id$
 #
diff -urNad openser-1.1.0~/scripts/openserctl.fifo openser-1.1.0/scripts/openserctl.fifo
--- openser-1.1.0~/scripts/openserctl.fifo	2006-07-04 19:49:58.000000000 +0200
+++ openser-1.1.0/scripts/openserctl.fifo	2006-07-12 14:24:36.382179242 +0200
@@ -1,4 +1,3 @@
-#!/bin/sh 
 #
 # $Id$
 #
diff -urNad openser-1.1.0~/scripts/openserctl.mysql openser-1.1.0/scripts/openserctl.mysql
--- openser-1.1.0~/scripts/openserctl.mysql	2006-07-04 19:49:58.000000000 +0200
+++ openser-1.1.0/scripts/openserctl.mysql	2006-07-12 14:24:25.445164760 +0200
@@ -1,4 +1,3 @@
-#!/bin/sh 
 #
 # $Id$
 #
diff -urNad openser-1.1.0~/scripts/openserctl.pgsql openser-1.1.0/scripts/openserctl.pgsql
--- openser-1.1.0~/scripts/openserctl.pgsql	2006-07-04 19:49:58.000000000 +0200
+++ openser-1.1.0/scripts/openserctl.pgsql	2006-07-12 14:24:07.066460035 +0200
@@ -1,4 +1,3 @@
-#!/bin/sh 
 #
 # $Id$
 #
diff -urNad openser-1.1.0~/scripts/openserctl.sqlbase openser-1.1.0/scripts/openserctl.sqlbase
--- openser-1.1.0~/scripts/openserctl.sqlbase	2006-07-04 19:49:59.000000000 +0200
+++ openser-1.1.0/scripts/openserctl.sqlbase	2006-07-12 14:24:02.376024976 +0200
@@ -1,4 +1,3 @@
-#!/bin/sh 
 #
 # $Id$
 #
diff -urNad openser-1.1.0~/scripts/openserctl.unixsock openser-1.1.0/scripts/openserctl.unixsock
--- openser-1.1.0~/scripts/openserctl.unixsock	2006-07-04 19:49:59.000000000 +0200
+++ openser-1.1.0/scripts/openserctl.unixsock	2006-07-12 14:23:59.847790472 +0200
@@ -1,4 +1,3 @@
-#!/bin/sh 
 #
 # $Id$
 #
